pywren/wren.py

```diff
@@ -6,6 +6,9 @@
     import pickle
 from tblib import pickling_support
 import logging
+import botocore
```

Comment:
No. Don't know how it shows up. Now almost every file in the project has unused imports. I'm planning to get rid off them in the PyLint PR, which is coming...very soon.